dev/release/verify-release-candidate.sh

```diff
@@ -347,18 +347,15 @@ install_csharp() {
 
   show_info "Ensuring that C# is installed..."
 
-  if which dotnet > /dev/null 2>&1; then
+  if dotnet --version | grep 7\.0 > /dev/null 2>&1; then
```

Comment:
what happens if `dotnet` is not installed? Does this script error out?

Reply:
The `if` fails and it downloads and installs a temporary local copy of .NET 7.